packages/ui-components/src/components/QrCode/QrCodeImage.tsx

```diff
@@ -18,13 +18,15 @@ interface QrCodeImageProps {
   humanReadableId: string;
   width?: number | undefined;
   margin?: number | string | undefined;
+  padding?: number | string | undefined;
```

Comment:
I wonder, since there are so many styles passed into this now, if it would be better to handle this with styled components in the respective parents? Either by creating a styled component of `QrCodeImage` or doing child styles on a wrapper around them. I don't really love the idea of passing in these things as props.